src/Http/Controllers/AuthorizationController.php

```diff
@@ -65,7 +65,7 @@ public function __construct(AuthorizationServer $server,
      * @param  \Illuminate\Http\Request  $request
      * @param  \Laravel\Passport\ClientRepository  $clients
      * @param  \Laravel\Passport\TokenRepository  $tokens
-     * @return \Illuminate\Http\Response
+     * @return \Laravel\Passport\Contracts\AuthorizationViewResponse|mixed|void
```

Comment:
According to the interface the `withParameters` method returns `mixed`

```suggestion
     * @return mixed
```

Reply:
Should return an instance of AuthorizationViewResponse. Will removed mixed/void.

Reply:
Not necessarily though, right? Because according to the interface the return type of `AuthorizationViewResponse` can be anything.

E.g. this would be valid:

```php
public function withParameters(...)
{
    return view('my-authorization-view');
}
```

Reply:
That would technically work as the controller would be able to return a response from the view however it would fail the return type definition of AuthorizationViewResponse. Following Fortify's precedence, the return type should be an implementation of the contract.

```
/**
 * Show the login view.
 *
 * @param  \Illuminate\Http\Request  $request
 * @return \Laravel\Fortify\Contracts\LoginViewResponse
 */
public function create(Request $request): LoginViewResponse
{
  return app(LoginViewResponse::class);
}
```

[     * @return \Laravel\Fortify\Contracts\LoginViewResponse](https://github.com/laravel/fortify/blob/9ba71f3e448ae44370bdfe72f19952e23b4d6191/src/Http/Controllers/AuthenticatedSessionController.php#L44)

Reply:
If that's the case then `withParameters()` should return `$this` in the interface rather than `mixed`, but IMHO the way you have it set up now makes more sense as it allows people to return whatever they want (and Laravel supports)

Reply:
`mixed` can never be combined with something else. I'd follow the contract.